erpnext/assets/doctype/asset_category/asset_category.js

```diff
@@ -4,7 +4,7 @@
 frappe.ui.form.on('Asset Category', {
 	onload: function(frm) {
 		frm.add_fetch('company_name', 'accumulated_depreciation_account', 'accumulated_depreciation_account');
-		frm.add_fetch('company_name', 'depreciation_expense_account', 'accumulated_depreciation_account');
+		frm.add_fetch('company_name', 'depreciation_expense_account', 'depreciation_expense_account');
```

Comment:
I think these two commits is not required. It's already merged (with some more changes). 

While git will handle this in most cases, it's better to avoid this. Do `git rebase -i HEAD~3` and drop them from your branch and force push.

Reply:
okay done